libs/domain/picking/offline/data-plugin.ts

```diff
@@ -20,37 +20,41 @@ export class OfflineDataPlugin extends ExecPlugin {
   protected subscription?: Subscription;
 
   constructor() {
-    super((app) => {
-      const injector = app!.requirePlugin(InjectionPlugin).getInjector();
+    super(() => undefined);
+  }
+
+  apply(app: App): void | Promise<void> {
```

Comment:
This plugin does not need to have logic in `apply()` method as it's a special `ExecPlugin` which accepts function in constructor and executes it automatically.
Otherwise you do not need to extend `ExecPlugin` and just implement `AppPlugin` interface.

Reply:
I wanted to leave it in constructor, but it wouldn't let me call it without also calling `super`, which is what I'm trying to avoid so we don't pull in all those other services we do not need.

Reply:
But you can call super with an empty function just like you are overriding the `apply()` method right now - it will have the same effect but with less code.